.github/workflows/tests.yml

```diff
@@ -84,13 +94,20 @@ jobs:
         echo "CGO_ENABLED=1" >> $GITHUB_ENV
       shell: bash
 
+    - name: Set test flags
+      run: |
+        if [ "${{ matrix.sonic }}" = "true" ]; then
+          echo "TEST_FLAGS=-tags sonic" >> $GITHUB_ENV
```

Comment:
Shouldn't this just be [GOFLAGS](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies) env var that's supported out of the box ?
Note that we should *add* to it, not override it